tracing/tracing-test.sh

```diff
@@ -238,48 +301,111 @@ check_jaeger_status()
 	[ "$errors" -eq 0 ] || die "errors still detected after $attempts attempts"
 }
 
+setup()
+{
+	start_jaeger
+
+	"${SCRIPT_PATH}/../.ci/configure_tracing_for_kata.sh" enable
+}
+
 run_test()
 {
-	local min_spans="$1"
-	local service="$2"
+	local service="$1"
+	local min_spans="$2"
+	local logdir="$3"
 
-	[ -z "$min_spans" ] && die "need minimum span count"
 	[ -z "$service" ] && die "need service name"
+	[ -z "$min_spans" ] && die "need minimum span count"
+	[ -z "$logdir" ] && die "need logdir"
 
-	create_trace
+	info "Running test for service '$service'"
 
-	check_jaeger_status "$service" "$min_spans"
+	logdir="$logdir/$service"
+	mkdir -p "$logdir"
 
-	check_missing_spans "$service" "$min_spans"
+	check_jaeger_status "$service" "$min_spans" "$logdir"
 
 	info "test passed"
 }
 
-main()
+run_tests()
 {
-	runtime_min_spans=10
-	shim_min_spans=5
+	# List of services to check
+	#
+	# Format: "name:min-spans"
+	#
+	# Where:
+	#
+	# - 'name' is the Jaeger service name.
+	# - 'min-spans' is an integer representing the minimum number of
+	#   trace spans this service should generate.
+	#
+	# Notes:
+	#
+	# - Uses an array to ensure predicable ordering.
```

Comment:
@jodh-intel do mean predicable here or predictable?

Reply:
That looks like a typo to me - I think it should say "predictable".